shell/platform/darwin/ios/platform_view_ios.h

```diff
@@ -98,6 +103,16 @@ class PlatformViewIOS final : public PlatformView {
     return platform_message_handler_;
   }
 
+  /**
+   * Gets the accessibility bridge created in this platform view.
+   */
+  AccessibilityBridge* GetAccessibilityBridge() { return accessibility_bridge_.get(); }
+
+  /**
+   * Handles accessibility message from accessibility channel.
+   */
+  void handleAccessibilityMessage(__weak id message, FlutterReply reply);
```

Comment:
```suggestion
  void HandleAccessibilityMessage(__weak id message, FlutterReply reply);
```
Need to fix up the call sites though.